src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+export interface TabType {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type TabListProps = {
+  tabs: TabType[];
+  handleTabClick: (tabItem: TabType) => void;
+  activeTab: TabType;
+};
+
+export const Tabs: React.FC<TabListProps> = ({
+  tabs,
+  handleTabClick,
+  activeTab,
+}) => {
+  return (
+    <>
+      <ul>
```

Comment:
The additional fragment is redundant here 
```suggestion
      <ul>
```